server/backend/quiz/views.py

```
@@ -62,10 +62,19 @@ def post(self, request):
 class UpdateDeleteQuizView(APIView):
     permission_classes: ClassVar = [IsAuthenticated, IsAdminUser]
 
-    def delete(self, request, category_id: int) -> Response:
-        category = get_object_or_404(Category, id=category_id)
-        category.delete()
-        return Response({"message": "Quiz deleted successfully"}, status=status.HTTP_200_OK)
+    def delete(self, request: Request) -> Response:
```

Ruff (0.8.2)

Undefined name `Request`

(F821)